go/runtime/host/host.go

```diff
@@ -32,6 +32,9 @@ type Config struct {
 
 	// LocalConfig is the node-local runtime configuration.
 	LocalConfig map[string]any
+
+	// Cleanup is an optional function that is called when the runtime is stopped.
+	Cleanup func()
```

Comment:
Would this be less hackish if it were moved to the sandbox config and behaved like `GetSandboxConfig`?

Reply:
Good idea, will change.

Reply:
Changed.